components/status-message/spec/StatusMessage.stories.mdx

```diff
@@ -51,8 +51,8 @@ A standard Status message.
 
 <Preview>
   <Story name="Standard">
-    <StatusMessage status="On hold" actions={[{ href: '#', text: 'Review' }]}>
-      - due for review 10 March 2018
+    <StatusMessage status="On hold" actions={[{ href: '#', text: 'CTA: Review' }]}>
```

Comment:
`Review`

(I think 'CTA' was just 'call to action'.)

Reply:
> _Central Technical Authority_
:shrug:

Reply:
Updated